src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx

```diff
@@ -132,21 +209,44 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
 
   render() {
     const istioRoute = kialiRoute('/istio');
+    const parsedRuleParams = this.parseRuleSearchParams();
+    let titleBreadcrumb: any[] = [];
```

Comment:
I like to split thinks out and I guess the breadcrumb generation it could be moved into another function. What do you think?

Reply:
Breadcrumb depends on the page, splitting would add a little bit more of complexity in the calls. IMO.

Reply:
Yep, depends on the page, sure. Moving it outside the render method but within the component would increase also the complexity?